modules/transport-netty4/src/main/java/org/elasticsearch/transport/netty4/Netty4Utils.java

```diff
@@ -61,7 +64,11 @@ public InternalLogger newInstance(final String name) {
     }
 
     public static void setup() {
-
+        // TODO remove this once we get a fix into Netty so we don't have to do this
+        AccessController.doPrivileged((PrivilegedAction<Void>) () -> {
+            ThrowableUtil.haveSuppressed();
```

Comment:
Would you leave a comment that this is triggering class initialization? This is horrific.

Reply:
I pushed b073780c01